src/app/token-options/mixins/multiple-section-new-due-time-mixin.ts

```diff
@@ -22,12 +22,12 @@ export function MultipleSectionNewDueTimeMixin<TBase extends Constructor<IGridVi
             this.registerDataPointSource(() =>
                 this.newDueTime instanceof MultipleSectionDateMatcher
                     ? {
-                          colName: 'New Assignment/Quiz Lock At (with exceptions)',
+                          colName: 'Change Canvas Assignment/Quiz’s Available Until to (with exceptions)',
```

Comment:
Please see comment on line 62 of `spend-for-assignment-resubmission-token-option-field-component-factory.ts`.